### DISCUSSION TITLE
Error using postgres

### DISCUSSION THREAD

**Discussion by sergiosilva-nsj** [Erudika/scoold#310] [Q&A] [answered]
Hello for everyone!

I'm trying use the Para with postgres (through a docker image). But, even adding the correct jars into "/para/lib" folder (into container, with -v option), the Para server returns the following error:

[ERROR] Error loading SQL driver "org.postgresql.Driver", class not found.

I'm using the jars: "para-dao-sql-1.45.4-shaded.jar" and "postgresql-42.4.0.jar". And I also tried put the "/para/lib" into CLASSPATH environment variable.

Ps: I also tried create a new docker image, using "erudikaltd/para", "erudikaltd/para-search-lucene" and "erudikaltd/para-dao-sql", but, in this case, occurs other two errors:

**First:**
`para_1      | 2022-06-09 19:45:28 [ERROR] Failed to initialize Para.
para_1      | java.lang.UnsupportedOperationException: Not implemented.
para_1      |   at com.erudika.para.core.search.MockSearch.findById(MockSearch.java:90)`

**Second:**
`2022-06-09 19:48:00 [ERROR] API request error: Not implemented.`

Can someone help me?

Thank you very much!

**Comment by albogdano** [comment, marked as answer]
The SQL plugin for Para is included in the Para JAR file so you don't need to add "para-dao-sql-1.45.4-shaded.jar". You still need to add the PostgreSQL driver though. Then you can try adding `-Dloader.path=lib` to the command line parameters when starting Para, or set this variable:
```
JAVA_OPTS="-Dloader.path=lib"
```
The Lucene plugin is also included in the default Para container (so you don't need to use Dockerfile-base).
The error you get is because your configuration file cannot be loaded. For more details look here: https://paraio.org/docs/#docker

**Reply by sergiosilva-nsj** [reply]
Thank you very much!
